quickstartproject/settings.py

```diff
@@ -121,6 +123,9 @@
 STATICFILES_DIRS = (str(BASE_DIR.joinpath('static')),)
 STATIC_URL = 'static/'
 
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'  
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
```

Comment:
I think staticfiles should be force removed from this repo and added to gitignore. See https://stackoverflow.com/questions/67486185/do-we-need-to-add-files-in-the-folder-that-white-noise-creates-to-the-gitignor

Reply:
@pamelafox Thanks for your feedback! I added your suggested change and then I added `staticfiles/` to the Django section of .gitignore in the latest push (I tested by running `py manage.py collectstatic` and making sure that the folder was excluded in VS Code. **Note** The `staticfiles` folder was never pushed up to GitHub in my Fork or the Azure-Samples repo.

Let me know if there's anything else.